File: llm_outputs/python_programs_corrected_by_llama/to_base.py
import string
def to_base(num, b):
    if num == 0 and b!= 10:
        return "0"
    result = ''
    alphabet = string.digits + string.ascii_lowercase
    while num > 0:
        i = num % b
        num = num // b
        result = alphabet[i] + result
    return result

File: llm_outputs/java_programs_corrected_by_llama/TO_BASE.java
package java_programs;

import java.util.*;

public class TO_BASE {

    public static String toBase(int num, int base) {
        if (base < 2 || base > 36) {
            throw new IllegalArgumentException("Invalid base");
        }

        StringBuilder result = new StringBuilder();
        String alphabet = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";

        do {
            int remainder = num % base;
            num /= base;
            result.insert(0, alphabet.charAt(remainder));
        } while (num!= 0);

        return result.toString();
    }
}